docs/getting-started.md

```diff
@@ -1 +1,106 @@
-This is a placeholder for some docs
+## 1. Overview
+Kubeflow is a novel open source tool for Machine Learning workflow orchestration on Kubernetes.
+It has great powers, but deploying it may not be so easy, depending on how and where you deploy
+your Kubernetes. This tutorial will show you an easy way to deploy Kubeflow using MicroK8s, a
+lightweight version of Kubernetes, in a few simple steps.
+
+### What you'll learn
+
+* How to deploy MicroK8s on Ubuntu
+* How to deploy Kubeflow using MicroK8s
+* How to access your Kubeflow dashboard
+
+### What you'll need
+* Desktop or Virtual Machine with Ubuntu 16.04 or above
+* A minimum of 4 CPU, 14GB RAM, 50GB Disk (recommended 8 CPU, 16GB RAM, 60GB Disk)
+* Some basic command-line knowledge
+* If installing on Windows or MacOS, see the MicroK8s docs for OS-specific requirements:
+  <https://microk8s.io/docs/install-alternatives> 
+
+
+## 2. Install MicroK8s
+
+To start, install MicroK8s, which is a minimal, lightweight Kubernetes you can run and use on
+practically any machine. It can be installed with a snap:
+
+1. Install Microk8s:
+
+       sudo snap install microk8s --classic
+
+For installation on other Operation Systems such as MacOS or Windows, see the alternative
+installation documentation here:
+
+https://microk8s.io/docs/install-alternatives
+
+
+## 3. Join admin group
+
+MicroK8s creates a group to enable seamless usage of commands which require admin privilege.
+
+1. To add your current user to the group and gain access to the `.kube` caching directory, run
+   the following two commands:
+
+       sudo usermod -a -G microk8s $USER
+       sudo chown -f -R $USER ~/.kube
+
+1. Exit and re-enter the session for permissions to take effect.
+
+
+## 4. Deployment
+
+Finally we are able to deploy all the Kubernetes services behind Kubeflow. To do this we just need
+to:
+
+1. Deploy Kubeflow using:
+
+       microk8s enable kubeflow
```

Comment:
This will not work due to: https://github.com/ubuntu/microk8s/issues/1396

Reply:
@knkski informed me the issue appears when you mix `sudo` invocations in the commands described here. As this is a known issue I mark this point resolved.